# HG changeset patch
# User Rik <rik@octave.org>
# Date 1405110972 25200
#      Fri Jul 11 13:36:12 2014 -0700
# Node ID c0fb31c46e87fdcd51888dd088ce3165295c4937
# Parent  82773ee8119a0568b19488630cb18ba06fb863cf
Change OpenGL dashed line style to 50% mark / 50% space.

*gl-render.cc (set_linestyle): Set stipple pattern to 0xF0F0 for
50% mark / 50% space rather than previous 75% mark / 25% space.

diff --git a/libinterp/corefcn/gl-render.cc b/libinterp/corefcn/gl-render.cc
--- a/libinterp/corefcn/gl-render.cc
+++ b/libinterp/corefcn/gl-render.cc
@@ -2920,17 +2920,17 @@ opengl_renderer::set_linestyle (const st
   if (s == "-")
     {
       glLineStipple (1, static_cast<unsigned short> (0xFFFF));
       solid = true;
     }
   else if (s == ":")
     glLineStipple (1, static_cast<unsigned short> (0x8888));
   else if (s == "--")
-    glLineStipple (1, static_cast<unsigned short> (0x0FFF));
+    glLineStipple (1, static_cast<unsigned short> (0xF0F0));
   else if (s == "-.")
     glLineStipple (1, static_cast<unsigned short> (0x020F));
   else
     glLineStipple (1, static_cast<unsigned short> (0x0000));
 
   if (solid && ! use_stipple)
     glDisable (GL_LINE_STIPPLE);
   else
